src/components/RenderRouter/HeroItem.tsx

```diff
@@ -277,19 +278,38 @@ class HeroItem extends React.Component<Props, State> {
             <div className="heroText2">{this.state.content.text6}</div>
             <div className="heroText2">{this.state.content.text7}</div>
             <div className="contactPastorLink">
-              {this.renderLinkButton(this.state.content.button1)}
-
-              {this.state.content.link1Text ? (
-                <div className="heroAContainer">
-                  <Link
-                    className="heroBlackBoxA inverted"
-                    to={this.state.content.link1Action}
-                  >
-                    {this.state.content.link1Text}
-                  </Link>
-                </div>
-              ) : null}
-              {this.state.content.addToCalendar ? (
+              {moment().weekday() === 0 && this.state.locationData.length ? ( // Is Sunday
+                <button
+                  onClick={() => this.navigateTo('/live')}
+                  className="calendarButton"
+                  style={{ paddingLeft: 0 }}
+                >
+                  <img
+                    className="calendarImage"
+                    src="/static/svg/Play.svg"
+                    alt="Contact Icon"
+                  />
+                  Watch Live
+                </button>
+              ) : moment().format('YYYY-MM-DD') === '2021-04-02' &&
+                this.state.locationData.length ? ( // Is good friday
+                <button
```

Comment:
Again `<LinkButton>` or `<Link>` is more appropriate.

Reply:
Thanks, Jon. I have made changes based on your suggestions. See 3e2040a937992cb02f1750c548651b2c11623bbb.
Let me know anything seems off.

Reply:
LGTM